### ISSUE TITLE
Does #33 make sense as-is given the 0.4 deprecration...

### ISSUE THREAD

**Issue by sbromberger** [johnmyleswhite/Style.jl#5]
of requiring explicit imports from Base for common operators?

See also https://groups.google.com/d/msg/julia-users/o3mfc0vfsz8/GhK-hckyCAAJ

That is, I'm not sure it should be considered a style violation to have something along the lines of

```
import Base: ==, +, -, /, *
```
in the main module file when you're defining (mulitple) methods for each of these.

**Comment by johnmyleswhite** [comment]
Nope, still as convinced as ever that using `import` is a bad idea.
